### PR TITLE
IOV Core 2.0.0 release

### LINKED ISSUE #1359: Release final version of IOV Core 2.0.0
Everthing is done from my side and this is ready to be released. It just needs a tag and be published to npm.

### PR DESCRIPTION
**Pull request by albertandrejev**
closes #1359.

### CODE REVIEW

**Comment by webmaster128** [comment]
2.0.1 is already tagged and on npm. Let's merge this.

There are already breaking changes on master, so I suggest creating a 2.0.2 on master and rename the 2.0.0 CHANGELOG entry to 2.0.2.

**Comment by davepuchyr** [comment]
> 2.0.1 is already tagged and on npm. Let's merge this.
> 
> There are already [breaking](https://github.com/iov-one/iov-core/pull/1394) [changes](https://github.com/iov-one/iov-core/pull/1392) on master, so I suggest creating a 2.0.2 on master and rename the 2.0.0 CHANGELOG entry to 2.0.2.

Abel is adding #1393 to the mix and will follow your suggestions.

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "lerna": "2.10.2",
-  "version": "2.0.0-alpha.7",
+  "version": "2.0.1",
```
Comment (davepuchyr) [inline lerna.json]: @webmaster128 , should `version` be `2.0.2` given the mess that we've created?  I'm not familiar with lerna.

Comment (webmaster128) [reply lerna.json]: Yes. Here you find all the info how to make the release (directly from master, no PR):

https://github.com/iov-one/iov-core/wiki/Publishing

Comment (webmaster128) [reply lerna.json]: `./node_modules/.bin/lerna publish` tags, pushes and publishes on npm. So if you did that, you're done and having a PR is pointless.

Comment (webmaster128) [reply lerna.json]: The add nonces step 3. ensures that all packages have a change (creation of an empty file), which ensures that all packages get the same version. If you don't do that, only packages with changes get a new version.

Comment (davepuchyr) [reply lerna.json]: Thank-you, Simon.

**Pull request merged**